Usiqee/Module/Account/AccountSettingsVC.swift

```diff
@@ -41,30 +47,61 @@ class AccountSettingsVC: UIViewController {
         setupView()
     }
     
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
```

Comment:
wa tu me crois jai jamais call le super de cet fonction mdr

Reply:
mdr moi je rajoute `super` systematiquement quand il ya un override mais j'ai checké, c'est useless en vrai `the default implementation of this method does nothing`la doc d'apple lol

Reply:
lolll